udata/core/dataset/events.py

```diff
@@ -40,35 +39,34 @@ def serialize_resource_for_event(resource):
 
 
 @task(route='high.resource')
-def publish(document, resource_id, action):
+def publish(url, document, resource_id, action):
     if action == KafkaMessageType.DELETED:
         resource = None
     else:
         resource = serialize_resource_for_event(get_by(document.resources, 'id', resource_id))
     message_type = f'resource.{action.value}'
-    produce(
-        kafka_uri=current_app.config.get('KAFKA_URI'),
-        topic=get_topic(message_type),
-        service='udata',
-        key_id=str(resource_id),
-        document=resource,
-        meta={'message_type': message_type, 'dataset_id': str(document.id)}
-    )
+    payload = {
+        'key': str(resource_id),
```

Comment:
I'm not sure about this payload. `key` doesn't make much sense now.
Maybe we should have something like
```
{
  'resource_id': '...',
  'dataset_id': '...',
  'document': '...'
}
```